### PR TITLE
Reorder peer review summary elements on the java side so it is valid …

### PR DESCRIPTION
**Pull request by amontenegro**
…agains the xsd

### CODE REVIEW

**Comment by amontenegro** [comment]
https://trello.com/c/C3j0Rm5B/2788-rc2-record-peer-review-review-group-id-is-an-invalid-element

File: orcid-model/src/main/java/org/orcid/jaxb/model/record/summary_rc2/PeerReviewSummary.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 import org.orcid.jaxb.model.record_rc2.ExternalIDs;
 
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(propOrder = { "createdDate", "lastModifiedDate", "source", "externalIdentifiers", "organization", "groupId", "completionDate" })
+@XmlType(propOrder = { "createdDate", "lastModifiedDate", "source", "externalIdentifiers", "completionDate", "groupId", "organization" })
```
Comment (rcpeters) [inline orcid-model/src/main/java/org/orcid/jaxb/model/record/summary_rc2/PeerReviewSummary.java]: @amontenegro should display-index play into this?

Comment (amontenegro) [inline orcid-model/src/main/java/org/orcid/jaxb/model/record/summary_rc2/PeerReviewSummary.java]: Hey @rcpeters, I dont know what you mean, display-index is an attribute(Already in the PeerReviewSummary object), and it don't affect the way we display the elements inside the peer review summary.

Comment (rcpeters) [inline orcid-model/src/main/java/org/orcid/jaxb/model/record/summary_rc2/PeerReviewSummary.java]: Ah I misunderstood never mind

**Pull request merged**